lib/oxrdf/src/literal.rs

```diff
@@ -33,10 +33,10 @@ use std::fmt::Write;
 /// );
 /// # Result::<_, LanguageTagParseError>::Ok(())
 /// ```
-#[derive(Eq, PartialEq, Debug, Clone, Hash)]
+#[derive(Eq, PartialEq, Debug, Clone, Hash, Ord, PartialOrd)]
 pub struct Literal(LiteralContent);
 
-#[derive(PartialEq, Eq, Debug, Clone, Hash)]
+#[derive(PartialEq, Eq, Debug, Clone, Hash, Ord, PartialOrd)]
```

Comment:
Aside: this is not the SPARQL order because it won't order e.g. integers by their values but by their serializations.